src/utils/requests.ts

```diff
@@ -0,0 +1,36 @@
+const NAV_LINKS = {
```

Comment:
@deadeagle63 - we have `SOCIAL_MEDIAS` defined in the constants.ts file. can you unify these so we dont have two copies?

Reply:
Combining